source/jsonnet/northern-ireland/lib/common_rules.libsonnet

```diff
@@ -1,4 +1,14 @@
 {
+  lastBirthdayAgeOver(age): {
+    id: 'age-last-birthday-answer',
+    condition: 'greater than or equal to',
+    value: age,
+  },
+  lastBirthdayAgeUnder(age): {
+    id: 'age-last-birthday-answer',
+    condition: 'less than',
+    value: age,
+  },
```

Comment:
Can combine these and use a flag instead but did this for readability.

Reply:
I think having the two is clearer